example/ios/KlaviyoReactNativeSdkExample/AppDelegate.mm

```diff
@@ -4,55 +4,97 @@
 
 @implementation AppDelegate
 
+// Change to NO if you don't want debug alerts or logs.
+BOOL isDebug = YES;
+
+
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   self.moduleName = @"KlaviyoReactNativeSdkExample";
-  // You can add your custom initial props in the dictionary below.
-  // They will be passed down to the ViewController used by React Native.
   self.initialProps = @{};
+  
+  // Installation Step 2: Set the UNUserNotificationCenter delegate to self
+  [UNUserNotificationCenter currentNotificationCenter].delegate = self;
 
-  [PushNotificationsHelper initializeSDK];
+  // Installation Step 3: Initilize the SDK with public key. 
+  [PushNotificationsHelper initializeSDK: @"YOUR_COMPANY_API_KEY_HERE"];
+  
+  // Installation Step 4: Request push permission from the user
   [PushNotificationsHelper requestPushPermission];
 
   return [super application:application didFinishLaunchingWithOptions:launchOptions];
 }
 
+// Installation Step 6: Implement this delegate to receive and set the push token
 - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
-    [PushNotificationsHelper setPushTokenWithToken:deviceToken];
+  // Installation Step 7: set the push token to Klaviyo SDK
+  [PushNotificationsHelper setPushTokenWithToken:deviceToken];
+  
+  if (isDebug) {
     NSString *token = [[deviceToken description] stringByTrimmingCharactersInSet:[NSCharacterSet characterSetWithCharactersInString:@"<>"]];
     token = [token stringByReplacingOccurrencesOfString:@" " withString:@""];
-
-    // Store or use the device token as needed
     NSLog(@"Device Token: %@", token);
+  }
 }
 
+// Installation Step 8: [Optional] Implement this if registering with APNs fails
 - (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error {
+  if (isDebug) {
     NSLog(@"Failed to register for remote notifications: %@", error);
+  }
 }
 
-// Handle push notifications when the app is in the background or terminated
+// Installation Step 9: Implement the delegate didReceiveNotificationResponse to handle push notifications when the app is in the background
```

Comment:
Clarifying question on this comment

> to handle push notifications when the app is in the background

This delegate method handles the user response to a notification right? like tapping on it?

Reply:
Yeah, it does. Were you thinking we should include that in as well? I do agree that the word "handle" is doing a lot of heavy lifting here.

Reply:
Updated, let me know.